modules/ingest-geoip/build.gradle

```diff
@@ -61,24 +61,6 @@ tasks.named("internalClusterTest").configure {
   }
 }
 
-tasks.named("thirdPartyAudit").configure {
```

Comment:
This is no longer needed, because this library is no longer used by geoip2. The jdk http client is now used by geoip2 library.